packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -128,6 +128,17 @@ describe("MafsGraph", () => {
         expect(line.getAttribute("y2")).toBe(-expectedY2 + "");
     });
 
+    it("renders X axis label with TeX and Y axis with text", () => {
+        const basePropsWithTexLabels = {
+            ...getBaseMafsGraphProps(),
+            labels: ["$1/2$", "1/2"],
+        };
+
+        render(<MafsGraph {...basePropsWithTexLabels} />);
+        expect(screen.getByText("1/2")).toBeInTheDocument();
+        expect(screen.getByText("\\text{1/2}")).toBeInTheDocument();
```

Comment:
I was a little confused when I first read this test because `1/2` appears in both labels. It's hard to tell which label goes with which assertion.

I might prefer splitting this into two tests:

```ts
    it("renders TeX in axis labels", () => {
        const basePropsWithTexLabels = {
            ...getBaseMafsGraphProps(),
            labels: ["$1/2$", "$3/4$"],
        };

        render(<MafsGraph {...basePropsWithTexLabels} />);
        expect(screen.getByText("1/2")).toBeInTheDocument();
        expect(screen.getByText("3/4")).toBeInTheDocument();
    });

    it("renders plain text in axis labels", () => {
        const basePropsWithTexLabels = {
            ...getBaseMafsGraphProps(),
            labels: ["1/2", "3/4"],
        };

        render(<MafsGraph {...basePropsWithTexLabels} />);
        expect(screen.getByText("\\text{1/2}")).toBeInTheDocument();
        expect(screen.getByText("\\text{3/4}")).toBeInTheDocument();
    });
```

Alternatively, I think just making the X and Y labels different would help.

Up to you what you want to do here.

Reply:
I was going back and forth between splitting them up and not splitting, but you're right, having separate tests to validate the functionality independently is probably the better move